keymanager-api-common/src/lib.rs

```diff
@@ -69,3 +72,28 @@ impl SignedPolicySGX {
         Ok(self.policy.clone())
     }
 }
+
+/// Returns the defult set of the trusted policy signers for key manager.
+pub fn default_trusted_policy_signers() -> TrustedPolicySigners {
```

Comment:
Why do we have this method, it seems dangerous? Shouldn't this be in the specific key manager only (e.g., in `simple-keymanager`)?

Reply:
yeah i just moved/renamed from ` keymanager-runtime/api/src/lib.rs` will put it into the `simple-keymanager-api` (and oasis-runtime keymanager)

Reply:
Sounds good. Also for the oasis-runtime one we should have an actual policy someday ;-)

Reply:
done